src/components/pages/Navbar/Navbar.js

```diff
@@ -43,6 +45,21 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isAuthenticated]);
 
+  const profile_id = user.profile_id;
+  const isMentor = user.role_id === 3;
+
+  // Determines the initial state of the Mentor toggle switch
+  useEffect(() => {
+    axiosWithAuth()
+      .post(`/profile/mentor-information`, { profile_id })
+      .then(res => {
+        const availability = res.data[0].availability;
+        setToggleStatus(availability);
+      })
+      .catch(err => console.log(err));
```

Comment:
This may need to be an issue on a greater scale, perhaps its own ticket: we need to handle these errors and I know that there are a few tickets that have been pushed lately using console.logs in place of error handling. It's something we need to keep track of/solve down the road.

Reply:
Definitely